angr/exploration_techniques/loop_seer.py

```diff
@@ -56,36 +58,20 @@ def setup(self, simgr):
             l.warning("LoopSeer uses normalized CFG. Recomputing the CFG...")
             self.cfg.normalize()
 
-        if type(self.functions) is str:
-            func = [self.cfg.kb.functions.function(name=self.functions)]
-
-        elif type(self.functions) is int:
-            func = [self.cfg.kb.functions.function(addr=self.functions)]
-
-        elif type(self.functions) is Function:
-            func = [self.functions]
+        func = []
```

Comment:
Nitpick: `func` is a singular noun. It's counter-intuitive to be a list. I would rename it to `funcs` or something plural.